reader_core/src/draw.rs

```diff
@@ -101,6 +105,42 @@ macro_rules! print_stat {
     };
 }
 
+macro_rules! print_pp {
```

Comment:
Lets convert this to a function since it isn't doing anything too funky that needs macro magic.